hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterJoinManager.java

```diff
@@ -115,8 +113,8 @@
     private final ConcurrentMap<UUID, Long> leftMembersUuids = new ConcurrentHashMap<>();
     private final long maxWaitMillisBeforeJoin;
     private final long waitMillisBeforeJoin;
-    private final long staleJoinPreventionDuration;
 
+    private long staleJoinPreventionDurationInMillis;
```

Comment:
```suggestion
    private final long staleJoinPreventionDurationInMillis;
```

I think the field can be `final` as long as it is initialized on `ClusterJoinManager` construction.